woocommerce/woocommerce-gateway-stripe
# chore: eslint jsx-handler-names rule

# Changes proposed in this Pull Request:

I sometimes see discussion in PRs about event handler/props names.
It would be great if some of those discussions could be avoided from the beginning by having a rule in place.
Luckily there is one! 😄 

This matches with React's conventions on event names.

# Testing instructions
- Event prop names should start with `on*`
- Event handlers can start with `handle` or `makeHandle` (for "Currying" patterns)
